.github/workflows/ci.yml

```diff
@@ -118,6 +120,7 @@ jobs:
           submodules: recursive
       - uses: asdf-vm/actions/install@v3
       - uses: bluefireteam/melos-action@v3
+      - uses: canonical/desktop-engineering/gh-actions/common/dpkg-install-speedup@main
       - run: sudo apt update
       - run: sudo apt install -y clang cmake curl gsettings-desktop-schemas libgtk-3-dev ninja-build pkg-config unzip
       - run: xvfb-run -a -s '-screen 0 1024x768x24 +extension GLX' flutter test integration_test/${{matrix.target}}
```

Comment:
I don't know if we use locales of the installed packages in the e2e tests, in which case we might not want to use the dpkg-install-speedup here.